digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/ServicePointBusStopDao.scala

```diff
@@ -0,0 +1,262 @@
+package fi.liikennevirasto.digiroad2.dao
+
+import fi.liikennevirasto.digiroad2.asset.{Modification, Property}
+import fi.liikennevirasto.digiroad2.dao.Queries.PropertyRow
+import slick.jdbc.{GetResult, PositionedResult, StaticQuery => Q}
+import fi.liikennevirasto.digiroad2.asset.PropertyTypes._
+import fi.liikennevirasto.digiroad2.asset.{MassTransitStopValidityPeriod, _}
+import fi.liikennevirasto.digiroad2.dao.Queries._
+import fi.liikennevirasto.digiroad2.service.pointasset.masstransitstop.MassTransitStopOperations
+import org.joda.time.{DateTime, LocalDate}
+import org.slf4j.LoggerFactory
+import fi.liikennevirasto.digiroad2.dao.Queries._
+import org.joda.time.DateTime
+import slick.driver.JdbcDriver.backend.Database
+import Database.dynamicSession
+import fi.liikennevirasto.digiroad2.Point
+import slick.jdbc.StaticQuery.interpolation
+import slick.jdbc.{GetResult, PositionedResult}
+import scala.language.reflectiveCalls
+import fi.liikennevirasto.digiroad2.Point
+import fi.liikennevirasto.digiroad2.asset.{LinkGeomSource, Modification, Property}
+import fi.liikennevirasto.digiroad2.dao.Queries.PropertyRow
+import fi.liikennevirasto.digiroad2.service.pointasset.masstransitstop.PersistedMassTransitStop
+import org.joda.time.LocalDate
+import org.slf4j.LoggerFactory
+import slick.jdbc.StaticQuery.interpolation
+import slick.jdbc.{GetResult, PositionedParameters, PositionedResult, SetParameter, StaticQuery => Q}
+import slick.driver.JdbcDriver.backend.Database
+import Database.dynamicSession
+import fi.liikennevirasto.digiroad2._
+import fi.liikennevirasto.digiroad2.asset.PropertyTypes._
+import fi.liikennevirasto.digiroad2.asset.{MassTransitStopValidityPeriod, _}
+import fi.liikennevirasto.digiroad2.dao.Queries._
+import fi.liikennevirasto.digiroad2.model.LRMPosition
+import fi.liikennevirasto.digiroad2.oracle.OracleDatabase
+import fi.liikennevirasto.digiroad2.service.pointasset.masstransitstop.{LightGeometryMassTransitStop, MassTransitStopOperations, MassTransitStopRow, PersistedMassTransitStop}
+import org.joda.time.{DateTime, Interval, LocalDate}
+import org.joda.time.format.ISODateTimeFormat
+import org.slf4j.LoggerFactory
+import scala.language.reflectiveCalls
+
+case class ServicePoint(id: Long, nationalId: Long, stopTypes: Seq[Int],
+                        municipalityCode: Int, lon: Double, lat: Double,
+                        created: Modification, modified: Modification, propertyData: Seq[Property])
+
+case class ServicePointRow(id: Long, externalId: Long, assetTypeId: Long, point: Option[Point],
+                           validFrom: Option[LocalDate], validTo: Option[LocalDate], property: PropertyRow,
+                           created: Modification, modified: Modification, municipalityCode: Int)
+
+class ServicePointBusStopDao {
```

Comment:
Let's try to extend this from MassTransitStopDao since they will use similar methods and types

Reply:
@marquesfm Applied suggestion